src/heretic/mn_menu.c

```diff
@@ -2220,7 +2220,8 @@ boolean MN_Responder(event_t * event)
 
     if (!MenuActive)
     {
-        if (key == key_menu_activate || gamestate == GS_DEMOSCREEN || demoplayback)
+        //[crispy] don't pop up the menu on other keys during a demo
+        if (key == key_menu_activate)
```

Comment:
Could you just comment out the other conditions in that line instead of entirely deleting them, please? Same in the Hexen code.